consensus/src/simplex/signing_scheme/ed25519.rs

```diff
@@ -275,7 +249,9 @@ impl signing_scheme::Scheme for Scheme {
 
         // Sort the signatures by signer index.
         entries.sort_by_key(|(signer, _)| *signer);
```

Comment:
No need to sort 👍

Reply:
We can leave a comment as to why we don't (we should leave a comment why we don't)